### ISSUE TITLE
authority of species

### ISSUE THREAD

**Issue by sjmeades** [Canadensys/vascan-data#1053]
[Originally posted on GoogleCode (id 828) on 2011-10-31]

<b>(This is the template to report a data issue for Vascan. If you want to</b>
<b>report another issue, please change the template above.)</b>

<b>What is the URL of the page where the problem occurs?</b>
http://data.canadensys.net/vascan/taxon/4721

<b>What data are incorrect or missing?</b>
Bulbostylis capillaris (Linnaeus) C.B. Clarke

<b>What data are you expecting instead?</b>
Bulbostylis capillaris (Linnaeus) Kunth ex C.B. Clarke

<b>If applicable, please provide an authoritative source.</b>
Tropicos; IPNI has Kunth in Hooker ex C.B.Clarke

**Comment by brouille** [comment]
[Originally posted on GoogleCode on 2011-11-15 14:47Z]

K. Gandhi: IPNI entry should be: Bulbostylis capillaris (L.) Kunth ex C.B.Clarke in Hook.

**Comment by brouille** [comment]
[Originally posted on GoogleCode on 2011-11-16 13:39Z]

It appears that the first impression was incorrect. The situation was more complex.

Here is the definite conclusion (modified from before): FNA is correct.  “B. capillaris (L.) C. B. Clarke” is OK. Eventually the Code will be corrected (i.e., remove the “ex” authorship).IPNI will be corrected.

Here is the reasoning:
In the pre-1994 Codes, it was “B. capillaris (L.) C. B. Clarke”.  That is how the type was conserved.
Only from the 1994 Tokyo Code onward, the “ex” authorship was added.
As I mentioned before, C.B. Clarke assigned the name to Kunth and gave a reference (Enum. 2: 205, 212),  but the latter did not coin the name B. capillaris.
Kunth (pp. 208-209) treated S. capillaris L. (1753) as a synonym of Isolepis barbata (confined to the Old World) and treated “S. capillaris L. (in Mantissa 1762; belongs to the New World)” as different.
It is perhaps for this reason, Fernald (Rhodora 40: 395. 1938), who designated the conserved type (from the New World), mentioned that C.B. Clarke’s ascription of the name to Kunth was an error.
I provided the above info to the Code Committee and asked why from the 1994 Tokyo Code onward, the “ex” authorship was added?
